src/rngs/small.rs

```diff
@@ -10,10 +10,10 @@
 
 use rand_core::{RngCore, SeedableRng};
 
+#[cfg(any(target_pointer_width = "32", target_pointer_width = "16"))]
```

Comment:
Why change it from `not(target_pointer_width = "64")`? Even with hypothetical 128-bit targets it's better to use Xoshiro128++ than to break builds. Alternatively, it's worth to at least add a `compile_error!` stating that we support only 16, 32, and 64 bit targets.

Reply:
1. Because this is explicit about what is supported and recommended
2. Effectively it is a compile error
3. I don't care about theoretical targets. If people want to target 16- or 128-bits on rand, they can submit patches.